src/compute.geometry/Program.cs

```diff
@@ -63,6 +63,10 @@ static void Main(string[] args)
                     int spanSeconds = int.Parse(span);
                     Shutdown.RegisterIdleSpan(spanSeconds);
                 });
+                x.AddCommandLineDefinition("rhinoSysDir", dir =>

```

Comment:
```suggestion
                x.AddCommandLineDefinition("rhinosysdir", dir =>

```

Keep case consistent with other CLI args?